### PR TITLE
Include information about disqualified nodes in the storage node payment report

### PR DESCRIPTION
**Pull request by mobyvb**
What: 
Add a column in the csv created by the storagenode usage command (`satellite reports storagenode-usage`) for the disqualified timestamp. If a node is not disqualified, this column of the row is empty. Otherwise, it will be the timestamp when the node was disqualified.

Still needs tests

Why:
https://storjlabs.atlassian.net/browse/V3-1944

Thanks for submitting a PR!

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Could the PR be broken into smaller PRs?
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?

### CODE REVIEW

**Comment by aligeti** [comment]
> What:
> Add a column in the csv created by the storagenode usage command (`satellite reports storagenode-usage`) for the disqualified timestamp. If a node is not disqualified, this column of the row is empty. Otherwise, it will be the timestamp when the node was disqualified.
> 
> Still needs tests
> 
> Why:
> https://storjlabs.atlassian.net/browse/V3-1944
> 
> Thanks for submitting a PR!
> 
> ## Code Review Checklist (to be filled out by reviewer)
> * [ ]  Does the PR describe what changes are being made?
> * [ ]  Does the PR describe why the changes are being made?
> * [ ]  Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
> * [ ]  Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
> * [ ]  Could the PR be broken into smaller PRs?
> * [ ]  Does the new code have enough tests? (_every_ PR should have tests or justification otherwise. Bug-fix PRs especially)
> * [ ]  Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
> * [ ]  Does any documentation need updating?

**Comment by ifraixedes** [comment]
@mobyvb I'm not sure if this branch can be deleted, because your are the owner could you check and remove it if it's the case?

**Comment by littleskunk** [comment]
I have marked this as a sprint release goal because it is the last release this month. At the end of the month we have to calculate the payout.

File: pkg/accounting/rollup/rollup_test.go [hunk 1]
```diff
@@ -26,8 +27,18 @@ type testData struct {
...
+		Reconfigure: testplanet.Reconfigure{
+			Satellite: func(log *zap.Logger, index int, config *satellite.Config) {
+				// 0 so that we can disqualify a node immediately by triggering a failed audit
+				config.Overlay.Node.AuditReputationLambda = 0
+			},
+		},
+	},
 		func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
+			dqedNodes, err := dqNodes(ctx, planet)
+			require.NoError(t, err)
```
[the remark this answers is not in the document]
Comment (ifraixedes) [reply pkg/accounting/rollup/rollup_test.go]: @mobyvb remember to also apply that assertion to the line 111

**Pull request merged**